src/ishikawa_backend.erl

```diff
@@ -44,15 +44,17 @@
 -include("ishikawa.hrl").
 
 -define(PEER_SERVICE, partisan_peer_service).
--define(WAIT_TIME_BEFORE_RESEND, 3000).
+-define(WAIT_TIME_BEFORE_CHECK_RESEND, 5000).
+-define(WAIT_TIME_BEFORE_RESEND, 10000).
```

Comment:
WAIT_TIME_BEFORE_CHECK_RESEND: the time to wait before check on all not acked messages in a process

WAIT_TIME_BEFORE__RESEND: the time to wait for resending a certain message if no acks where received
In the second one we only resend a message if time_of_check - time_previous_send > WAIT_TIME_BEFORE__RESEND